eclair-core/src/main/scala/fr/acinq/eclair/wire/FailureMessage.scala

```diff
@@ -83,7 +83,7 @@ object FailureMessageCodecs {
     .typecase(UPDATE | 12, (("amountMsat" | uint64) :: ("channelUpdate" | channelUpdateWithLengthCodec)).as[FeeInsufficient])
     .typecase(UPDATE | 13, (("expiry" | uint32) :: ("channelUpdate" | channelUpdateWithLengthCodec)).as[IncorrectCltvExpiry])
     .typecase(UPDATE | 14, (("channelUpdate" | channelUpdateWithLengthCodec)).as[ExpiryTooSoon])
-    .typecase(UPDATE | 20, (("flags" | binarydata(2)) :: ("channelUpdate" | channelUpdateWithLengthCodec)).as[ChannelDisabled])
+    .typecase(UPDATE | 20, (("flags" | binarydata(1)) :: ("channelUpdate" | channelUpdateWithLengthCodec)).as[ChannelDisabled])
```

Comment:
That's not compliant, the `UPDATE|20` message still includes the 2-bytes flags.

Reply:
Do I have to change value back to `2` there?

Reply:
Yes, that's what BOLT 4 says